productMods/js/UIModules/pageElements/SelectorAddField.js

```diff
@@ -0,0 +1,37 @@
+
+
+var SelectorAddField = function(configData){
+	
+	this.container = html.form("/vivo/newInstance", configData.id)
```

Comment:
please make the path universal: if the webapp.name is not vivo, this script does not work.